Compilation error with Fujitsu compilers

When I try to compile this code with the Fujitsu compilers, I encounter an error.
```
/opt/FJSVxtclanga/tcsds-1.2.40/bin/frtpx  -I/home/ra000009/data/u10035/devel/toml-f/build/include  -c /home/ra000009/data/u10035/devel/toml-f/src/tomlf/ser.f90 -o CMakeFiles/toml-f-lib.dir/src/tomlf/ser.f90.o
Fortran diagnostic messages: program name(tomlf_ser)
 Module subprogram name(toml_serialize)
  jwd1997i-s  "/home/ra000009/data/u10035/devel/toml-f/src/tomlf/ser.f90", line 114, column 18: Stop code must be scalar default character constant expression or scalar integer constant expression in STOP or ERROR STOP statement.
make[2]: *** [CMakeFiles/toml-f-lib.dir/build.make:439: CMakeFiles/toml-f-lib.dir/src/tomlf/ser.f90.o] Error 1
make[2]: Leaving directory '/vol0004/ra000009/data/u10035/devel/toml-f/build'
make[1]: *** [CMakeFiles/Makefile2:363: CMakeFiles/toml-f-lib.dir/all] Error 2
make[1]: Leaving directory '/vol0004/ra000009/data/u10035/devel/toml-f/build'
make: *** [Makefile:146: all] Error 2
```
The subroutine in question is:
```
function toml_serialize(val, config) result(string)
   !> TOML value to visit
   class(toml_value), intent(inout) :: val 

   !> Configuration for serializer
   type(toml_ser_config), intent(in), optional :: config

   !> Serialized JSON value
   character(len=:), allocatable :: string

   type(toml_error), allocatable :: error

   call toml_dumps(val, string, error, config=config)
   if (allocated(error)) then
      error stop error%message                                                                                   
   end if
end function toml_serialize
```
I believe that Fujitsu's compiler may be correct here in regards to the Fortran 2008 standard. The stop code must be either a `scalar-default-char-constant-expr` or `scalar-int-constant-expr` (see page 188 of https://j3-fortran.org/doc/year/10/10-007r1.pdf). This would not be the case for error%message.  Of course, Fujitsu's compilers have been known to have bugs...

If my interpretation is correct, maybe I could submit a pull request where the error is simply printed out and error stop is called without a code. Thanks and let me know what you think.

--- a/src/tomlf/ser.f90
+++ b/src/tomlf/ser.f90
@@ -111,7 +111,8 @@ function toml_serialize(val, config) result(string)
 
    call toml_dumps(val, string, error, config=config)
    if (allocated(error)) then
-      error stop error%message
+      print '(a)', "Error: " // error%message
+      error stop 1
    end if
 end function toml_serialize
 

